### PR TITLE
Fixed "Gift Card system architecture" links

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [ ] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [X] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

Fixed "Gift Card system architecture" links

### CODE REVIEW

File: docs/guides/Integration Guides/gift-card-integration-guide/gift-card-integration-guide-system-architecture.md [hunk 1]
```diff
@@ -9,26 +9,26 @@ updatedAt: "2020-10-14T23:59:34.329Z"
...
 ![Gift Card Hub](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/Integration%20Guides/gift-card-integration-guide/gift-card-integration-guide-system-architecture-0_10.png)
 
-This article presents an overview of this architecture, so you understand its building blocks and how they interact with each other. We will also explain the purpose of the [Gift Card API](https://developers.vtex.com/reference/giftcard-api-overview) and the [Gift Card Hub API](https://developers.vtex.com/reference/giftcard-hub-api-overview) and how they should be used.
+This article presents an overview of this architecture, so you understand its building blocks and how they interact with each other. We will also explain the purpose of the [Gift Card API](https://developers.vtex.com/docs/api-reference/giftcard-api#overview) and the [Gift Card Hub API](https://developers.vtex.com/docs/api-reference/giftcard-hub-api#overview) and how they should be used.
 [block:callout]
 {
   "type": "info",
```
Comment (karenkrieger) [inline docs/guides/Integration Guides/gift-card-integration-guide/gift-card-integration-guide-system-architecture.md:15]: Ajeitar o formato do callout

File: docs/guides/Integration Guides/gift-card-integration-guide/gift-card-integration-guide-system-architecture.md [hunk 2]
```diff
@@ -9,26 +9,26 @@ updatedAt: "2020-10-14T23:59:34.329Z"
...
+This is the protocol defining the communication standards gift card providers must follow in their integration with Gift Card Hub. The [Gift Card Provider Protocol](https://developers.vtex.com/docs/api-reference/giftcard-provider-protocol#overview) section of our API Reference provides details on the eleven endpoints that should be exposed by the middleware connecting the external gift card provider to VTEX.
 
 ## VTEX Gift Card Provider
 
-Our native gift card provider is bundled into your VTEX store, and [Gift Card API](https://developers.vtex.com/reference/giftcard-api-overview) is the implementation of the Gift Card Provider Protocol for it. It can be used to manage VTEX gift cards directly - although Gift Card Hub API provides the same functionality in a more generic abstraction layer.
+Our native gift card provider is bundled into your VTEX store, and [Gift Card API](https://developers.vtex.com/docs/api-reference/giftcard-api#overview) is the implementation of the Gift Card Provider Protocol for it. It can be used to manage VTEX gift cards directly - although Gift Card Hub API provides the same functionality in a more generic abstraction layer.
 [block:callout]
```
Comment (karenkrieger) [inline docs/guides/Integration Guides/gift-card-integration-guide/gift-card-integration-guide-system-architecture.md:32]: Ajustar callout

**Review by karenkrieger — CHANGES_REQUESTED** [review]

File: docs/guides/Integration Guides/gift-card-integration-guide/gift-card-integration-guide-system-architecture.md (hunk 1 above)
Comment (ricardoaerobr) [reply docs/guides/Integration Guides/gift-card-integration-guide/gift-card-integration-guide-system-architecture.md]: Corrigido.

File: docs/guides/Integration Guides/gift-card-integration-guide/gift-card-integration-guide-system-architecture.md (hunk 2 above)
Comment (ricardoaerobr) [reply docs/guides/Integration Guides/gift-card-integration-guide/gift-card-integration-guide-system-architecture.md]: Corrigido.

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**